language.md

```diff
@@ -539,8 +539,8 @@ TODO: Have a way of destructuring so that the parent is at fault if it doesn't s
 - (nominal typing)
 
 metaWrapper (String value) [
-  Doc: "documentation",
-  ToText: { value -> ... },
+Doc: "documentation",
+ToText: { value -> ... },
```

Comment:
This indentation should probably stay

```suggestion
  Doc: "documentation",
  ToText: { value -> ... },
```

Reply:
Oh, seems like the formatter automatically dedented it because it wasn't in a code block.